packages/api/package.json

```diff
@@ -36,9 +36,9 @@
       "!tests/endpoints/granules/helpers.js",
       "!tests/helpers/**/*.js"
     ],
-    "fail-fast": true,
     "verbose": true,
-    "timeout": "20m"
+    "timeout": "30m",
```

Comment:
What's the reasoning here, particularly given `failFast` was already set to `true`?

Reply:
Oh great catch. This was a holdover from testing and a mistake to leave in. I was messing with the timeout to see if that was causing failures but never set it back.

https://github.com/nasa/cumulus/pull/3577/commits/dc645d2f3896a9962383ed2ff2f1f4511fef867e